code/blake2/Makefile

```diff
@@ -4,33 +4,43 @@ HACL_HOME=$(realpath ../..)
 # test, a Low* test, or just a binary archive (like libcurve.a).
 all: dist/Makefile.basic
 
-test: all
+test: all dist/blake2-test.exe
 	dist/blake2-test.exe
 
+test32: all dist/blake2-test32.exe
+	dist/blake2-test32.exe
+
 # Defines rules for producing .checked, .krml, .depend, etc.
 include $(HACL_HOME)/Makefile.local
 
-CFLAGS += -I$(HACL_HOME)/lib/c -mavx -mavx2
-export CFLAGS
-
 # CUSTOMIZE HERE: how to produce binary objects
 # An archive with all the compiled code in this directory.
 
+BLAKE2_BUNDLE= -bundle Hacl.Blake2b_32=Hacl.Impl.Blake2.\* \
+  -bundle Hacl.Blake2s_32=Hacl.Impl.Blake2.\* \
+  -bundle Hacl.Blake2s_128=Hacl.Impl.Blake2.\* \
+  -bundle Hacl.Blake2b_256=Hacl.Impl.Blake2.\*
+
+
 dist/Makefile.basic: $(filter-out %/prims.krml,$(ALL_KRML_FILES))
 	$(KRML) $^  $(BASE_FLAGS) $(BLAKE2_BUNDLE) \
-	  -funroll-loops 8 \
+	  -funroll-loops 12 \
 	  -fbuiltin-uint128 \
 	  -tmpdir dist \
+	  -skip-compilation \
 	  -ccopts -std=gnu11,-g,-O3 \
-	  -I$(KRML_HOME)/include \
-	  -I$(HACL_HOME)/lib/c" \
-	  -add-include '"krml/internal/target.h"' \
+	  -I "../../../karamel/include" \
```

Comment:
This seems to assume that karamel is installed in the same folder as hacl-star. That seems like a strong assumption (which doesn't hold on my machine for instance), what is the rationale for dropping the use of KRML_HOME?